### DISCUSSION TITLE
Alt+Tab when in Full Screen on Gnome Host, Windows Guest.

### DISCUSSION THREAD

**Discussion by MCCMikey** [rustdesk/rustdesk#8869] [Feature Request]
Alt-Tab is not passed to the guest when the host is running on Fedora, Gnome. Instead it functions on the local computer as it would while in any app. 

Tiger VNC on the same system will pass Alt+Tab to the guest if the session is in full screen mode but not in windowed mode. This seems a good way to handle this feature.

**Comment by rustdesk** [comment]
@fufesou make sense, but not high priority.

**Reply by MCCMikey** [reply]
Thanks for considering it. Just tried with NoMachine in the same environment. There it's handled as a checkbox on "Input" labeled "Grab the Keyboard Input".

**Reply by MCCMikey** [reply]
Adding to this, I've found it's dependent on which windowing system you're using on the Linux host. If I select 'GNOME on Xorg' at the login screen it works perfectly. It doesn't work on Wayland. 

So, if your Windows key or your alt+tab isn't working on a Windows Guest with a Linux Host, maybe try switching windowing environments if possible.
